src/core/io/pla_parser.cpp

```diff
@@ -29,20 +39,26 @@ namespace syrec {
                 }
 
                 if (inout.size() != 2)
-                    throw std::invalid_argument("Invalid I/O");
+                    throw std::invalid_argument("Expected exactly 2, received " + std::to_string(inout.size()));
```

Comment:
Expected two .. what? there's something missing in that error message.

Reply:
done 👍🏼